.circleci/config.yml

```diff
@@ -257,9 +257,15 @@ commands:
           environment:
             GORELEASER_VERSION: 0.177.0
           command: |
-            curl -sfL -o goreleaser-install https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh && \
-              sh goreleaser-install -b ${GOPATH}/bin v${GORELEASER_VERSION} && \
-              rm goreleaser-install
+            # checksum from `checksums.txt` file at https://github.com/goreleaser/goreleaser/releases
+            GO_RELEASER_SHA="8dd5fff1d04eff3789d200920bf280391f96fd5cc1565dd0d6e0db2b9a710854"
```

Comment:
Can this be moved up next to the GORELEASER_VERSION in the `environment:` stanza? That way the two variables pertaining to the goreleaser version are under the same maintenance scheme.